package.json

```diff
@@ -15,16 +17,20 @@
     "injection-js": "^2.1.0",
     "lodash": "^4.17.4",
     "merge-graphql-schemas": "^1.2.0",
-    "pem": "^1.12.3"
+    "moment": "^2.19.1",
+    "pem": "^1.12.3",
+    "result-class": "^1.2.0",
+    "winston": "^2.4.0"
   },
   "devDependencies": {
     "@types/aws-serverless-express": "^2.1.1",
     "@types/body-parser": "^1.16.5",
     "@types/compression": "^0.0.34",
     "@types/cors": "^2.8.1",
     "@types/lodash": "^4.14.76",
-    "@types/node": "^8.0.44",
+    "@types/node": "7.0.43",
```

Comment:
New @types/node has some conflict with other lib, so fallback to a old one. As aws lamba support node 6.10.3 only.